src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -16,47 +12,36 @@
 
 namespace Microsoft.Docs.Build
 {
-    public enum MarkdownPipelineType
-    {
-        ConceptualMarkdown,
```

Comment:
seems previous `markdown/inlinemarkdown` attribute doesn't resolve moniker zone,  will this brings perf impact or result diff?

Reply:
That seems to me like a bug @928PJY markdown/inline markdown is also used in SDP, we should also handle moniker zones in SDP right?

Reply:
we don't have the plan to support moniker zone syntax in SDP markdown before, so it hasn't been added to the inlineMarkdown pipeline, but seems it will just bring perf impact to the repos which contain moniker zone syntax in SDP files, to keep the consistency of markdown pipelines and since it matches our future goal, we can add it to inlineMarkdown pipeline now.